code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/interactions/UIEvent.kt

```diff
@@ -61,8 +61,8 @@ sealed interface UIEvent<T : AepUITemplate, S : AepCardUIState> {
      *
      * @param T represents UI template model associated like [SmallImageTemplate], which backs the composable on which the event has occurred.
```

Comment:
Wondering if we can change Interact to have these fields

1. aepUi
2. action of type `UIAction` which would have different interact enums like `CLICK`, `SWIPE` etc.
3. id: Might be useful to look up in schema data? not too sure
4. actionUrl of type String. Since every clickable element's model (eg: AepButton) has actionUrl guessing it should be easy to add here